client/src/components/App.jsx

```diff
@@ -48,7 +46,7 @@ const App = () => {
       </div>
       <div><RelatedItems productId={currentId} /></div>
       <div className="questions-container"><QuestionsAndAnswers productId={currentId} /></div>
-      <div><ReviewList /></div>
+      <div id="reviews"><ReviewList /></div>
```

Comment:
Why did you add id reviews to reviewList for kevin's component?

Reply:
@afong888 My Star Ratings review link should point to the Reviews & Ratings section so I added an id so that my link could find it. The link is in ProductInfo.jsx line 49.
FYI @kyang9021